config/settings/docker.py

```diff
@@ -23,7 +23,8 @@
 # The optional vars will set the SERVER_EMAIL information as needed
 OPTIONAL_ENV_VARS = ['AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY', 'AWS_SES_REGION_NAME',
                      'AWS_SES_REGION_ENDPOINT', 'SERVER_EMAIL', 'SENTRY_JS_DSN', 'SENTRY_RAVEN_DSN',
-                     'REDIS_PASSWORD', 'DJANGO_EMAIL_BACKEND'] + SMTP_ENV_VARS
+                     'REDIS_PASSWORD', 'DJANGO_EMAIL_BACKEND',
+                     'POSTGRES_HOST'] + SMTP_ENV_VARS
```

Comment:
I put this here because it's an optional environment variable, but I'm totally willing to move this to `ENV_VARS` with the rest of the Postgres-related variables.

Reply:
I think this is an okay place to put it, since it is optional.